### ISSUE TITLE
SymbolKeyTestBase.ResolveAndVerifySymbol ignores 'newCompilation' input parameter

### ISSUE THREAD

**Issue by mavasani** [dotnet/roslyn#45894]
See https://github.com/dotnet/roslyn/pull/45847/commits/6e0b6778e3fa44441c5cd1d6297bb3708a4be3be#diff-b21c98ba01bccb58ada91d6c25ac9422R32

Test helper has an unused `newCompilation` parameter. Should `ResolveAndVerifySymbol` calls below use `newCompilation`? We should either remove or respect this parameter.

**Comment by mavasani** [comment]
@CyrusNajmabadi

**Comment by CyrusNajmabadi** [comment]
yes.  taht seems like a bug. ResolveAndVerifySymbolList should not take originalCompilation.  It should only take newCompilation and pass that on isntead.

**Comment by CyrusNajmabadi** [comment]
This was fixed.
